src/server/types.ts

```diff
@@ -116,7 +116,7 @@ export interface Route<Data = any> {
 
 // --- APP ---
 
-export interface AppProps {
+export interface AppProps extends PageProps {
```

Comment:
`data` is always `unknown` I guess?

Reply:
Yup, `type` parameter is fine as well for `data` property

Reply:
Is not `unknown`, is `any`.